python/lsst/summit/extras/focusAnalysis.py

```diff
@@ -406,25 +416,25 @@ class NonSpectralFocusAnalyzer:
     focusAnalyzer.run() can be used instead of the last two lines separately.
     """
 
-    def __init__(self, embargo=False):
+    def __init__(self, embargo: bool = False):
         self.butler = makeDefaultLatissButler(embargo=embargo)
         self._bestEffort = BestEffortIsr(embargo=embargo)
 
     @staticmethod
-    def gauss(x, *pars):
+    def gauss(x: float, *pars: "float"):
         amp, mean, sigma = pars
         return amp * np.exp(-((x - mean) ** 2) / (2.0 * sigma**2))
 
     def run(
         self,
-        dayObs,
-        seqNums,
+        dayObs: int,
+        seqNums: List[int],
         *,
-        manualCentroid=None,
-        doCheckDispersed=True,
-        doDisplay=False,
-        doForceCoM=False,
-    ):
+        manualCentroid: Tuple[float] | None = None,
```

Comment:
I think we can be more specific and make this a two-ple of floats, _i.e._ `(float, float)`, however that should be expressed, as it'll always be of length two.